bundle/manifests/operator.ibm.com_commonaudits.yaml

```diff
@@ -19,148 +19,147 @@ spec:
-          type: string
-        metadata:
-          type: object
-        spec:
-          description: CommonAuditSpec defines the desired state of CommonAudit
-          properties:
-            clusterIssuer:
-              type: string
-            enabled:
-              description: EnableAuditLoggingForwarding defines if audit logs should be forwarded to an SIEM or not
-              type: boolean
-            fluentd:
-              description: CommonAuditSpecFluentd defines the desired state of Fluentd
-              properties:
-                imageRegistry:
-                  type: string
-                pullPolicy:
-                  type: string
-                resources:
-                  description: ResourceRequirements describes the compute resource requirements.
-                  properties:
-                    limits:
-                      additionalProperties:
-                        anyOf:
-                        - type: integer
-                        - type: string
-                        pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
-                        x-kubernetes-int-or-string: true
-                      description: 'Limits describes the maximum amount of compute resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/'
-                      type: object
-                    requests:
-                      additionalProperties:
-                        anyOf:
-                        - type: integer
-                        - type: string
-                        pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
-                        x-kubernetes-int-or-string: true
-                      description: 'Requests describes the minimum amount of compute resources required. If Requests is omitted for a container, it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/'
-                      type: object
-                  type: object
-              type: object
-            issuer:
-              type: string
-            outputs:
-              description: CommonAuditSpecOutputs defines the configurations for forwarding audit logs to Splunk or Syslog
-              properties:
-                hostAliases:
-                  items:
-                    description: CommonAuditSpecHostAliases defines the host alias for an SIEM
+  versions:
+  - name: v1alpha1
```

Comment:
This needs to be `v1` since that is the version it was at before this change. You can see the version on the original line 159